README.md

```diff
@@ -6,7 +6,7 @@ Java projects for the Google Cloud Platform.
 # Google Cloud Platform Java Dependency Dashboard
 
 [Google Cloud Platform Java Dependency Dashboard](
-https://storage.googleapis.com/cloud-opensource-java-dashboard/dashboard/dashboard.html)
+https://storage.googleapis.com/cloud-opensource-java-dashboard/com.google.cloud/libraries-bom/snapshot/index.html)
```

Comment:
Probably doesn't matter a great deal, but this maybe should be a separate PR just for docs since this URL won't be immediately live when you commit and the docs will be.